assets/styles/styles.css

```diff
@@ -79,9 +79,6 @@ select {
 .preloader-wrapper {
```

Comment:
I think you'll want a place or structure for "src" CSS. This could be if you want to use postcss for things, or use sass. I think most people that are using just basic CSS regret it as some point because code organization and keeping things dry. This, again, depends on your app, but I'm just taking a guess.